app/index.html

```diff
@@ -7,9 +7,13 @@
   <link rel="stylesheet" href="emoji/emoji-20px.css">
   <link rel="stylesheet" href="emoji/emoji-45px.css">
   <link rel="stylesheet" href="styles/main.css">
-  <!-- Once we change our actual l10n framework to l10n.js (or better, l20n.js),
-       we need to merge fxosrate strings to our new l10n files -->
-  <link rel="resource" type="application/l10n" href="components/fxosRate/locales/locales.ini" />
```

Comment:
we need to decide if we translate the fxosrate files (add them to the properties file) or if we pull the translations from the fxosrate repro. there are only 3 languages there.

We can generate in transifex a separate fxosrate properties file and let our  translators translate them. and do a pull request for fxosrate so they will have more translations in there repro